tests/containers/rootless_podman.pm

```diff
@@ -36,14 +36,15 @@ sub run {
 
     install_podman_when_needed($host_distri);
     allow_selected_insecure_registries(runtime => $runtime);
-    my $user      = $testapi::username;
-    my $check_msg = 'Checking allocation range of user';
-    script_run "grep $user /etc/subuid || echo /etc/subuid has no uid range for $user", output => $check_msg;
-    script_run "grep $user /etc/subgid || echo /etc/subgid has no gid range for $user", output => $check_msg;
-    assert_script_run "usermod --add-subuids 200000-201000 --add-subgids 200000-201000 $user";
+    my $user         = $testapi::username;
+    my $subuid_start = get_user_subuid($user);
+    record_soft_failure 'bsc#1179261 - YaST creates incomplete user accounts' if $subuid_start eq '$user';
```

Comment:
How is possible that `$subuid_start eq '$user'`?
I thought that `$subuid_start` is integer and `$user` string.

What does 'incomplete user account' mean?
Does this mean that the `/etc/subuid` might be broken?

Reply:
> What does 'incomplete user account' mean?

That's the title of the bug report

> Does this mean that the /etc/subuid might be broken?

Incomplete, yes

Reply:
Unresolving, the first question wasn't answered yet

Reply:
cc @b10n1k